fixtures/attribute-behavior/AttributeTableSnapshot.md

```diff
@@ -5052,7 +5077,7 @@
 | Test Case | Flags | Result |
 | --- | --- | --- |
 | `href=(string)`| (changed)| `"https://reactjs.com/"` |
-| `href=(empty string)`| (changed)| `"http://localhost:3000/"` |
+| `href=(empty string)`| (initial, warning)| `<empty string>` |
```

Comment:
Logs `An empty string ("") was passed to the href attribute. To fix this, either do not render the element at all or pass null to href instead of an empty string.`. 

This seems like a bug since `<a href={null} />` would also not set the href to the base url. `<a href="" />` is a pretty common "Home" link I believe.

Reply:
I believe this was introduced in https://github.com/facebook/react/pull/26379. @sebmarkbage could you clarify if this is intended?

Reply:
It's a bug. Will work on a fix.